src/Controller/SantaController.php

```diff
@@ -88,7 +88,7 @@ public function participants(Request $request, string $application): Response
         $errors = [];
 
         if ($request->isMethod('POST')) {
-            $selectedUsers = $request->request->get('users', []);
+            $selectedUsers = $request->request->all('users', []);
```

Comment:
This does not work like this.  There is no default parameter anymore. it must be `$request->request->all()['users'] ?? []`

Reply:
Indeed, I removed the useless parameters.